### PR TITLE
Requesting changes in permission section

### PR DESCRIPTION
**Pull request by sunainaTA**
[InputbufferChanges.docx](https://github.com/MicrosoftDocs/sql-docs/files/10672307/InputbufferChanges.docx)
HI Team,

I have added an attachment to describe one use-case scenario specific to the permission part of this article and what steps were taken to resolve the issue. Please review and let us know if the mentioned changes can be made to improve the documentation.

### CODE REVIEW

File: docs/t-sql/database-console-commands/dbcc-inputbuffer-transact-sql.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+[InputbufferChanges.docx](https://github.com/MicrosoftDocs/sql-docs/files/10672254/InputbufferChanges.docx)
```
Comment (Court72) [inline docs/t-sql/database-console-commands/dbcc-inputbuffer-transact-sql.md]:
```suggestion
<!-- [InputbufferChanges.docx](https://github.com/MicrosoftDocs/sql-docs/files/10672254/InputbufferChanges.docx) -->

```

**Comment by Court72** [comment]
@rwestMSFT 

Can you review the proposed changes and update the article if needed? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by rwestMSFT** [comment]
@sunainaTA We do not accept Word documents as attachments. Please make your edits inline, in Markdown.

You can learn about contributing to Microsoft Docs in the [contributor guide](https://learn.microsoft.com/en-us/contribute/).

#please-close

**Comment by rwestMSFT** [comment]
@sunainaTA Just following up to say we'll open an internal issue to track this. Thank you for your feedback.

**Pull request closed without merging**